### PR TITLE
fix:overflow on filters dropdown on small viewports

### PR DESCRIPTION
**Pull request by rajeshj11**
/claim #8840 
## How did you test this change?
demo video:
https://www.loom.com/share/fb44dc3a8f8c43a289b49316ea836680

### CODE REVIEW

**Comment by rajeshj11** [comment]
@Vadman97 please review

**Comment by rajeshj11** [comment]
@Vadman97 could you please take a look at it?

**Comment by rajeshj11** [comment]
> @Vadman97 could you please take a look at it?

^^

File: frontend/src/components/Search/SearchForm/SearchForm.css.ts [hunk 1]
```diff
@@ -184,7 +184,23 @@ export const comboboxGroup = style([
...
-
 export const toggle = style({
 	paddingLeft: '8px',
 })
+
+export const showAllResults = style({
+	cursor: 'pointer',
+	display: 'inline-block',
+	alignItems: 'center',
+	padding: '0 10px',
```
Comment (Vadman97) [inline frontend/src/components/Search/SearchForm/SearchForm.css.ts:195]: nice, looks like a significant improvement

before

Screenshot 2024-07-11 at 17 31 03

after

Screenshot 2024-07-11 at 17 29 56

@rajeshj11 is it possible to clean up the spacing between `show all results` and the filter? looks like in the vertical dimension the rows are too close together.

**Review by Vadman97 — APPROVED** [review]

File: frontend/src/components/Search/SearchForm/SearchForm.css.ts (hunk 1 above)
Comment (rajeshj11) [reply frontend/src/components/Search/SearchForm/SearchForm.css.ts:195]: Ack

**Comment by rajeshj11** [comment]
@Vadman97 resolved the feedback changes. please reivew

**Comment by rajeshj11** [comment]
> @Vadman97 resolved the feedback changes. please review

sample view
Screenshot from 2024-07-12 01-33-06

**Comment by rajeshj11** [comment]
@Vadman97 could please take a look at the updated changes

**Comment by rajeshj11** [comment]
@Vadman97 Are there any changes needed on this? please let me know

**Comment by rajeshj11** [comment]
@Vadman97 could please block sometime to review these changes

**Pull request merged**